### PR TITLE
radicale: 3.0.5 -> 3.0.6

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/radicale/3.x.nix [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
+  pname = "Radicale";
+  version = "3.0.6";
 
-  # No tests in PyPI tarball
-  src = fetchFromGitHub {
-    owner = "Kozea";
-    repo = "Radicale";
-    rev = version;
-    sha256 = "0w8qkjm7b62cr49dbis41kvv3179sfmvvzlhlc0rbqss6vmwbq4p";
+  src = python3.pkgs.fetchPypi {
```
Comment (SuperSandro2000) [inline pkgs/servers/radicale/3.x.nix:7]: Do they ship the tests on pypi again?

Comment (dotlambda) [reply pkgs/servers/radicale/3.x.nix:7]: Yes

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109504` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>etesync-dav: log <a href="https://termbin.com/dl7u">https://termbin.com/dl7u</a></li>
  </ul>
</details>
<details>
<summary>
1 package built:
</summary>
<ul>
<li>radicale (radicale3)</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109504` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>etesync-dav</li>
<li>radicale (radicale3)</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Can you mark etesync-dav broken on darwin?

**Comment by hmenke** [comment]
Result of `nixpkgs-review pr 109504` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>etesync-dav</li>
    <li>radicale (radicale3)</li>
  </ul>
</details>

**Review by hmenke — APPROVED** [review]

**Pull request merged**